ml-agents-envs/mlagents/envs/communicator_objects/custom_action_pb2.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file should not be here, it should be with the others in `ml-agents/ml-agents-envs/mlagents/envs/communicator_objects`

Reply:
You mean the ones in ml-agents/mlagents right? I removed those three - but I think this one should be here.